### PR TITLE
[Keyboard] Whale 75

### PR DESCRIPTION
**Pull request by ebastler**
## Description

Adds support for the Linworks Whale75 keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Adds support for Linworks Whale75 keyboard

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/linworks/whale75/whale75.c [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+/* Set lock LEDs to their states*/
+bool led_update_kb(led_t led_state) {
+    if (led_update_user(led_state)) {
+        writePin(A8, !led_state.num_lock);
+        writePin(A9, !led_state.caps_lock);
+        writePin(A13, !led_state.scroll_lock);
+    }
+    return true;
+}
```
Comment (zvecr) [inline keyboards/linworks/whale75/whale75.c:37]:
```suggestion

```

File: keyboards/linworks/whale75/config.h [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+#define MATRIX_COLS 16
+
+#define MATRIX_ROW_PINS { B3, B4, B5, B6, B7, A0 }
+#define MATRIX_COL_PINS { A1, A2, A3, A4, A5, A6, A7, B0, B1, B2, B10, B11, B12, B13, B14, B15 }
+#define UNUSED_PINS { C13, C14, C15 }
+
+/* COL2ROW, ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+#define DEBOUNCE 5
+
```
Comment (zvecr) [inline keyboards/linworks/whale75/config.h:41]:
```suggestion

#define LED_NUM_LOCK_PIN A8
#define LED_CAPS_LOCK_PIN A9
#define LED_SCROLL_LOCK_PIN A13
#define LED_PIN_ON_STATE 0

```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/linworks/whale75/config.h [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+#define BACKLIGHT_LEVELS 5
+#define BACKLIGHT_ON_STATE 1
+#define BACKLIGHT_PWM_OUTPUT_FREQUENCY 1000 // Increases backlight PWM freq if compiled with an unmerged PR. Does no harm without it.
+
+/* Underglow */
+#define RGB_DI_PIN B9
+#define WS2812_EXTERNAL_PULLUP // This board uses a pull-up + 5 V tolerant GPIO in open drain config ro generate a 5 V signal
+#define RGBLED_NUM 18
+/* section for PWM WS2812 driver */
+#define RGB_DI_PIN B9
```
Comment (drashna) [inline keyboards/linworks/whale75/config.h:62]: You have this defined twice
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/linworks/whale75/config.h (hunk 3 above)
Comment (ebastler) [reply keyboards/linworks/whale75/config.h]: ooops. thanks!

File: keyboards/linworks/whale75/info.json [hunk 4]
```diff
@@ -0,0 +1,369 @@
+{
+  "keyboard_name": "Lx3 Whale 75",
+  "keyboard_folder": "linworks/whale75",
+  "maintainer": "ebastler",
+  "width": 16.25,
+  "height": 6.5,
+  "layouts": {
+    "LAYOUT_ansi": {
+      "key_count": 80,
```
Comment (fauxpark) [inline keyboards/linworks/whale75/info.json:9]:
```suggestion
```

File: keyboards/linworks/whale75/info.json [hunk 5]
```diff
@@ -0,0 +1,369 @@
...
+        { "x": 3.75, "y": 5.25, "w": 6.25},
+        { "x": 10, "y": 5.25, "w": 1.5},
+        { "x": 11.5, "y": 5.25, "w": 1.5},
+        { "x": 13.25, "y": 5.25, "w": 1},
+        { "x": 14.25, "y": 5.25, "w": 1},
+        { "x": 15.25, "y": 5.25, "w": 1}
+      ]
+    },
+    "LAYOUT_ansi_wkl": {
+        "key_count": 79,
```
Comment (fauxpark) [inline keyboards/linworks/whale75/info.json:99]:
```suggestion
```

File: keyboards/linworks/whale75/info.json [hunk 6]
```diff
@@ -0,0 +1,369 @@
...
+          { "x": 3, "y": 5.25, "w": 7},
+          { "x": 10, "y": 5.25, "w": 1.5},
+          { "x": 11.5, "y": 5.25, "w": 1.5},
+          { "x": 13.25, "y": 5.25, "w": 1},
+          { "x": 14.25, "y": 5.25, "w": 1},
+          { "x": 15.25, "y": 5.25, "w": 1}
+        ]
+    },
+    "LAYOUT_iso": {
+        "key_count": 81,
```
Comment (fauxpark) [inline keyboards/linworks/whale75/info.json:188]:
```suggestion
```

File: keyboards/linworks/whale75/info.json [hunk 7]
```diff
@@ -0,0 +1,369 @@
...
+          { "x": 3.75, "y": 5.25, "w": 6.25},
+          { "x": 10, "y": 5.25, "w": 1.5},
+          { "x": 11.5, "y": 5.25, "w": 1.5},
+          { "x": 13.25, "y": 5.25, "w": 1},
+          { "x": 14.25, "y": 5.25, "w": 1},
+          { "x": 15.25, "y": 5.25, "w": 1}
+        ]
+    },
+    "LAYOUT_iso_wkl": {
+        "key_count": 80,
```
Comment (fauxpark) [inline keyboards/linworks/whale75/info.json:279]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/linworks/whale75/readme.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
+# Lx3 Whale 75
```
Comment (Erovia) [inline keyboards/linworks/whale75/readme.md:1]: Can you make this to be more in line with our [current template](https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/readme.md)?

**Review by Erovia — CHANGES_REQUESTED** [review]

**Comment by ebastler** [comment]
Thanks @Erovia - I modified the readme.md to fit the template. I am still waiting for a picture from the keyboard designer, which he will send me on monday, I think. I pushed the file the way it is (without a link) for now.

**Review by drashna — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by ebastler** [comment]
Readme fixed - it follows the example now.

**Review by fauxpark — APPROVED** [review]

**Pull request merged**